contents/blog/the-posthog-array-1-39-0.mdx

```diff
@@ -0,0 +1,128 @@
+---
+date: 2022-08-29
+title: 'Array 1.39.0: The persons on events beta'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: PostHog 1.39.0 introduces a new beta for you to try, a new display chart and big improvements to our client libraries.
+---
+
+Want to know more about what we're up to? [Subscribe to HogMail, our newsletter](/newsletter), which we send out every two weeks!
+
+> Running a self-hosted instance? Check out our [guide to upgrading PostHog](/docs/runbook/upgrading-posthog).
+
+## PostHog 1.39.0 release notes
+
+**Release highlights:**
+
+- Beta: [Persons on events](#beta-persons-on-events)
+- [New: Format your insights to show durations](#new-format-your-insights-to-show-durations)
+- [New: Big, bold number chart type](#new-big-bold-number-chart-type)
+- [Improved: Turbo properties](#improved-turbo-properties)
+- [Improved: History log now shows changes to insights](#improved-history-log-now-shows-changes-to-insights)
+- [Improved: Client and server-side library improvements](#improved-client-and-server-side-library-improvements)
+- [Improved: Automatically comment SQL queries](#improved-automatically-comment-sql-queries)
+- [Apps: Census integration](#apps-census-integration)
+
+### Beta: Persons on events
```

Comment:
Is this too technical for a title? Should we do something funnier/more exciting?

Reply:
I second this. I think for most readers who have no context on this change or on how our data is structured this won't stand out as much as it should.

Reply:
Open to suggestions here, as it's a bit of a complex thing to parse anyway (persons not people), but I've tweaked this to make the benefit clearer to users: `Beta: Improving query performance by combining persons and events`